s4e-web/cypress/support/commands.js

```diff
@@ -136,3 +136,92 @@ Cypress.Commands.add('hasMailWithSubject', {prevSubject: true}, (mails$, subject
 Cypress.Commands.add('hasMailTo', {prevSubject: true}, (mails$, recipient) =>
   cy.wrap(mails$).filterByRecipient(recipient).should('not.have.length', 0)
 );
+
+Cypress.Commands.add('getMailBySubject', subject => {
+  cy.getAllMails().filterBySubject(subject).should('have.length', 1);
+});
+
+Cypress.Commands.add('clickActivateLink', () => {
```

Comment:
I wouldn't keep such commands in here. The method is used only in specific case of register. I'd recommend to keep in  register method and use it in tests by calling it.

Reply:
Cypress commands should be created only generic purpose, like it was with mail fetching commands.